jrqs-app/src/java/ua/dp/isd/jrqs/SqlBuilder.java

```diff
@@ -69,7 +67,12 @@ public void addCommaSeparatedParameters(String commaSeparatedParameters) {
             return;
         }
 
-        sqlParameters = commaSeparatedParameters.toLowerCase().split(",");
+        String[] splitted = commaSeparatedParameters.toLowerCase().split(",");
+        if (splitted.length > MAX_PARAMETERS) {
+            throw new IllegalArgumentException("The number of maximum allowed arguments have been exceeded.");
```

Comment:
Sounds reasonable. Though can you move this check closer to UI layer? Basically because it is business rule. The logic that does work with SQL shouldn't know about this restriction. You can pass string[] to this method, for example and limit number of this array before it has been passed. Just a thought.